### PR TITLE
Methods Proposal

### PR DESCRIPTION
**Pull request by murrayrodney**
Updated the methods section and added a couple of the graphs we talked about to the Summary Statisitics section

### CODE REVIEW

**Comment by murrayrodney** [comment]
I think we'll end up with some merge conflicts in the summary statistics section. I think the easiest thing will be for you to review this when you get a chance @brstanle, and merge your PR after you've looked at the comments I made. Then, I can pull those changes back into here and address the conflicts.

File: reports/proposal.md [hunk 1]
```diff
@@ -5,14 +5,40 @@
...
 
 ## Proposed Methods
-* Response and predictor variables
-* What analyses will be conducted
-* How will assumptions be checked
-* Results to be reported
-* Other details
+For analysis of the visualization data, the response variable will be a binary categorical value (0/1) indicating if the quadrant was correctly identified by the subject.  The remaining variables summarized below will be used in a Generalized Linear Mixed Model (GLMM) with the assumption that the response follows a binomial distribution.
+
+| Variable Name | Type (continuous/categorical)                           | Effect Type      |
```
Comment (brstanle) [inline reports/proposal.md:40]: Yea this is better!

File: reports/proposal.md [hunk 2]
```diff
@@ -5,14 +5,40 @@
...
-* etc.
\ No newline at end of file
+
+Below is a plot demonstrating how the centroid for many of the images was identified as belonging to Quadrant B resulting in a large imbalance and need for potential correction.
+
+<img src="./figures/Provided_Quadrant.png" alt="Provided Quadrant Mismatch" style="zoom:50%;" />
+
+Below are some plots of numerical variables which have been binned for the continuous cases to illustrate how the proportion correct may change over different values of each variable. 
+
+<img src="./figures/approximate_proportions.png" alt="Approximate proportions for binned values of continuous variables" style="zoom:50%;" />
```
Comment (brstanle) [inline reports/proposal.md]: Was `./figures` dir get added to repo?

File: reports/proposal.md [hunk 3]
```diff
@@ -5,14 +5,40 @@
...
 * Box plots for subject accuracy
-* Binned variables showing approximate probabilities
-* etc.
\ No newline at end of file
+
+Below is a plot demonstrating how the centroid for many of the images was identified as belonging to Quadrant B resulting in a large imbalance and need for potential correction.
+
+<img src="./figures/Provided_Quadrant.png" alt="Provided Quadrant Mismatch" style="zoom:50%;" />
+
+Below are some plots of numerical variables which have been binned for the continuous cases to illustrate how the proportion correct may change over different values of each variable.
```
Comment (brstanle) [inline reports/proposal.md]: Not sure I follow this sentence

**Review by brstanle — APPROVED** [review]

File: reports/proposal.md (hunk 2 above)
Comment (murrayrodney) [reply reports/proposal.md]: Nope, good catch!

**Pull request merged**